lib/Quantization/Quantization.cpp

```diff
@@ -296,6 +296,14 @@ static Node *quantizeNode(Function *F, Node *node,
     quantizedNode = F->createSigmoid(SN->getName(), quantizedInputs[0]);
     break;
   }
+  case Kinded::Kind::SplatNodeKind: {
+    auto *SPN = cast<SplatNode>(node);
```

Comment:
please add a test to end2end quantization test to cover splat node.

Reply:
I think it would be better to implement this as a graph optimization by merging quantization node into splat. This potentially will cover more scenarios for splat -> quantize sequences.

Reply:
i just realize the code here is useless, as the splat would still be floating op as i dont have the quantized type for the splat here.

Reply:
maybe i just disable the support for SplatNode in isOpSupported() ?